### PR TITLE
Merge branch remove super user password logging 20190312

### PR DESCRIPTION
**Pull request by kamsonline**

### CODE REVIEW

File: generators/app/templates/README.md [hunk 1]
```diff
@@ -95,6 +95,10 @@ To turn auto-bootstrap on/off set the `bootstrap` flag to true/false in the appr
 
 For more information on how the above command works see the migrations section.
 
+## Activate and login (Local)
+
+User can be automatically logged in automatically after successful activation. This is managed by the config `autoLoginAfterActivate` which is enabled by default. Set this config to false if you do not prefer the user to be logged in automatically after successful activation.
```
Comment (afcastano) [inline generators/app/templates/README.md]: Little typo here: "User can be automatically logged in automatically"

File: generators/app/templates/client/src/modules/accounts/Activate.tsx [hunk 2]
```diff
@@ -46,53 +51,35 @@ const ConfirmReset: React.SFC<Props> = ({ classes, match, history }) => (
...
-            <form onSubmit={handleSubmit}>
-              <Field
-                label="Name"
-                fullWidth
-                name="name"
-                margin="normal"
-                validate={required('Name is required')}
-                component={Input}
-              />
+    <Form onSubmit={activate(client, match.params.code, () => history.push('/'))}>
```
Comment (afcastano) [inline generators/app/templates/client/src/modules/accounts/Activate.tsx]: @kamsonline This is the Activate call that was changed from a Graphql call to a Rest endpoint?

**Comment by afcastano** [comment]
This PR is contained in #11 . Should we close this one and only merge #11 @kamsonline

File: generators/app/templates/README.md (hunk 1 above)
Comment (kamsonline) [reply generators/app/templates/README.md]: I will fix the typo

File: generators/app/templates/client/src/modules/accounts/Activate.tsx (hunk 2 above)
Comment (kamsonline) [reply generators/app/templates/client/src/modules/accounts/Activate.tsx]: Yes, this calls the rest api.

**Pull request merged**